library/build.gradle

```diff
@@ -13,6 +13,7 @@ ext.tangram_version = "0.4"
 apply plugin: 'com.android.library'
 apply plugin: 'checkstyle'
 apply plugin: 'com.github.dcendents.android-maven'
+apply plugin: 'com.neenbedankt.android-apt'
```

Comment:
Oh cool. This is the same plugin I have used in the past to get Dagger generated classes to resolve in IntelliJ.

I didn't realize it added another dependency scope as well.